### ISSUE TITLE
Intent link blocked by browser

### ISSUE THREAD

**Issue by Esteban26** [google/model-viewer#702]
### Description
Recently we created some pens at codepen to show some examples about the AR functionality. But we faced some issues on android versions while model viewer tries to open Scene viewer using the intent link.

By default fullscreen is enabled, but scene viewer is never launched, I tested the same example outside the pen and it works perfectly, 

After checking the browser console at  browser stack I noticed this warning was appearing while clicking the AR button.

Navigation is blocked: intent://googlewebcomponents.github.io/model-viewer/examples/assets/Astronaut.glb.........

It seems that model viewer is not able to open intent links if it is within an iframe.

#### Live Demo
https://codepen.io/cylindo/pen/aeGdBY

### Browser Affected

- [X ] Chrome version 74.0.3729.136
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [X ] Android 7 - Device Galaxy S8
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### Versions
https://googlewebcomponents.github.io/model-viewer/dist/model-viewer.js

**Comment by soraryu** [comment]
Think this is the same as #693 which broke model-viewer everywhere over the last days but is (hopefully) resolved with latest hotfix 0.5.1 as of yesterday. When did you encounter the issues, and have you tried again today?

**Comment by Esteban26** [comment]
Thanks for your quick response @soraryu, I updated to use the https://unpkg.com/@google/model-viewer/dist/model-viewer.js version so It is already using 0.5.1 but i think it is another issue since I'm still getting the same console warning.

**Comment by Esteban26** [comment]
@soraryu This is a video where you can see the messages I'm getting.

https://drive.google.com/file/d/1W1hjBFwwO3kYPsdWSis6xoyIx63O4O4S/view?usp=sharing

**Comment by cdata** [comment]
@Esteban26 that message is shown if you try to trigger an intent from inside an `<iframe>`. I believe that this is prohibited by Chrome.

**Comment by Esteban26** [comment]
yeah, thats what I though, so, is it possible it is fixed on chrome if we report it? Or it won't be fixed?

**Comment by cdata** [comment]
I think there is a chance it could be fixed if you report it. If you can get a conversation going on crbug.com, I would be able to explain the use case to the folks involved internally.

**Comment by Esteban26** [comment]
Done, https://bugs.chromium.org/p/chromium/issues/detail?id=992940

**Comment by cdata** [comment]
Thanks! I'll keep an eye on the bug and participate as it seems helpful.

Separately, I Google'd this issue and discovered an old Stack Overflow on the topic: https://stackoverflow.com/questions/16428102/loading-chrome-android-intents-from-an-iframe

In it, @paulkinlan suggests that the rationale is that there is no good pattern for navigating to a fallback URL when intenting from inside an `<iframe>`. I can definitely see how this _might_ be a problem, although I wonder if it's more a technical challenge or a UX one.

**Comment by Esteban26** [comment]
Ok, just a comment, I updated the ticket, where I'm facing the issue is on Chrome version 74.0.3729.136 with Android 7, same chrome version with Android 9 works fine.

**Comment by Esteban26** [comment]
This is the Android 9 video https://drive.google.com/file/d/1amSdf-IZrZJxBfTmTgh4L1pf1Ste3UZj/view?usp=sharing
